### PR TITLE
removing as these obstruct deployments

### PR DESCRIPTION
**Pull request by fire-ant**

### CODE REVIEW

File: pkg/processor/crd/crd.go [hunk 1]
```diff
@@ -21,11 +21,6 @@ const crdTeml = `apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: %[1]s
-  annotations:
-    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "%[2]s.fullname" . }}-%[4]s
```
Comment (arttor) [inline pkg/processor/crd/crd.go:25]: Sorry, but we cannot remove cert-manager annotation. It is used for webhook validation in several projects.

**Review by arttor — CHANGES_REQUESTED** [review]

File: pkg/processor/crd/crd.go (hunk 1 above)
Comment (fire-ant) [reply pkg/processor/crd/crd.go:25]: another optional? -crd-full-metadata

Comment (arttor) [reply pkg/processor/crd/crd.go:25]: I suggest to check if source CRD contains `cert-manager.io/inject-ca-from` annotation and remove it only if not.

**Comment by arttor** [comment]
fixed in #47

**Pull request closed without merging**